### ISSUE TITLE
DRM related tests race on network latency

### ISSUE THREAD

**Issue by SingingTree** [w3c/web-platform-tests#4135]
The DRM tests have network dependencies that result in races. Independent of if the tests are access locally or at http://w3c-test.org/ they need to make requests to the license server, so there is always some variable lag due to this. However, even in relatively straight forward tests, such as [drm-events](https://github.com/w3c/web-platform-tests/blob/master/encrypted-media/drm-events.html) this appears to result in rare intermittent fails when running locally (for my case a license request must round tripped from New Zealand).

I've been looking at adjusting some of the test timeouts, but it's a bit of a whack a mole. I'm raising this issue so further discussion can be had around how to best handle this.

**Comment by cpearce** [comment]
I suggest all .\*drm.\* tests have their timeout extended across the board. Then we don't need to keep playing whack-a-mole and up the timeouts as we trip over intermittent failures caused by latency with the DRM license server.

**Comment by ddorwin** [comment]
@mwatson2, any thoughts?

**Comment by jdsmith3000** [comment]
I guess the question is: Are they reasons to limit these tests to 10 second execution limits?  The only downside might be a bit longer execution time on valid timeout results.  @cpearce; Do you have an alternate time in mind (e.g. 20s)?

**Comment by ddorwin** [comment]
Yes, that would be my main concern. Looking at the latest results, we don't have many timeouts, so I think this should be rare and thus okay. (We want tests to fail rather than timeout if possible. See #3938. In some cases, such as depending on an event, it is not possible.)

@SingingTree or @cpearce, please prepare a PR.

**Comment by mwatson2** [comment]
The test harness offers two choices of timeout: short (10s) or long (60s). We'll need to add the following to every HTML file:
```
<meta name="timeout" content="long">
```
